src/docs/glossary/index.md

```diff
@@ -1,7 +1,56 @@
 ---
 title: Glossary
-author: Santiment Team
+author: Ivan
 ---
 
-- [Asset](/glossary/asset/)
-- [Slug](/glossary/slug/)
+Find the meaning of terms used throughout the documentation
+
+- [asset](#asset)
+- [slug](#slug)
```

Comment:
If you want to add all terms on that page - remove pages for `asset` and `slug`

Reply:
The asset here leads to its own page (no detailed description here). I could move all things in this page but it would become huge

Reply:
okay, agree